textproc/pandoc/Portfile

```diff
@@ -24,33 +25,43 @@ long_description    \
     Textile, groff man pages, plain text, Emacs Org-Mode, AsciiDoc, EPUB (v2 \
     and v3), FictionBook2, and S5, Slidy and Slideous HTML slide shows.
 
-depends_lib-append  \
-                    port:hs-aeson \
-                    port:hs-alex \
-                    port:hs-attoparsec \
-                    port:hs-base64-bytestring \
-                    port:hs-blaze-html \
-                    port:hs-blaze-markup \
-                    port:hs-data-default \
-                    port:hs-extensible-exceptions \
-                    port:hs-happy \
-                    port:hs-highlighting-kate \
-                    port:hs-hslua \
-                    port:hs-http \
-                    port:hs-mtl \
-                    port:hs-network \
-                    port:hs-pandoc-types \
-                    port:hs-parsec \
-                    port:hs-random \
-                    port:hs-syb \
-                    port:hs-sha \
-                    port:hs-tagsoup \
-                    port:hs-temporary \
-                    port:hs-texmath \
-                    port:hs-text \
-                    port:hs-unordered-containers \
-                    port:hs-vector \
-                    port:hs-xml \
-                    port:hs-yaml \
-                    port:hs-zip-archive \
-                    port:hs-zlib
+depends_build-append  \
+                    port:stack
+
+use_configure       no
+
+# the PATH environment must provide /usr/bin/gcc, not ${prefix}/bin/gcc
+build.env \
+    PATH=/usr/bin:$env(PATH) \
+    PREFIX=${prefix} \
+    CC=${configure.cc}
```

Comment:
Cool, now that `configure.cc` works, do you still need to override `build.env`? What happens if you simply remove `build.env`? (Try it locally first. I'm not behind my mac at the moment to be able to test.)

I would imagine that macports already sets all these three variables correctly.

Reply:
Just to clarify:
* buildbot is only running the builds after the changes end up in repository (https://build.macports.org)
* for pull requests we use Travis and Azure
  * if Azure succeeds, you may ignore Travis as Travis too often fails for irrelevant reasons

Reply:
It's highly likely that it already works if you say
```
sudo port -v install pandoc configure.compiler=macports-clang-8.0
```
for example. Or at least it should if everything was set up correctly.

Blacklisting gcc compiler if it doesn't work is another thing. (There's absolutely no need to fix anything in that respect at the moment.)